examples/nidaqmx/dsa-shared-timebase-and-trig-analog-input-and-output.py

```diff
@@ -0,0 +1,395 @@
+# This example configures a total of four tasks on two PXI Dynamic Signal Acquistion (DSA) devices.
+# Each device runs a task for analog input and another for analog output. This example illustrates
+# continuous analog input and output operations.
+#
+# Instructions for Running:
+#    1. Select which type of Synchronization method you want to use.
+#    2. Select the physical channel to correspond to where your
```

Comment:
"to correspond to where your signal is input" -> "corresponding to the input signal"?

From whence it corresponds?

I assume this is copied from the C API doc, so I'm OK with it as is. (And the rest of it is phrased really well. Just this line got overcooked).

Reply:
Yeah, I like your suggestion. (and yeah, I did copy the docs over, but I did fix a few typos along the way, so might as well fix this too!)